src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1279,6 +1283,14 @@ private void ThrowIfOperationInProgress(Task operationTask, [CallerMemberName] s
                 throw new InvalidOperationException(SR.Format(SR.net_Websockets_AlreadyOneOutstandingOperation, methodName));
             }
         }
+        
+        private void ThrowIfDisposed()
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(GetType().FullName);
```

Comment:
In Windows implementation, we use `GetType().FullName`, and I copy the pattern from there.

https://github.com/dotnet/corefx/blob/f81984bcd0843e05e51a9280573ec67a28fc41f4/src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/WebSocketBase.cs#L2409